mmv1/products/cloudbuild/terraform.yaml

```diff
@@ -65,6 +65,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           installation_id: 31300675
           pat_secret: '"projects/gcb-terraform-creds/secrets/github-pat/versions/latest"'
           repo_uri: '"https://github.com/gcb-repos-robot/tf-demo.git"'
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloudbuild_trigger_bitbucket_server"
+        primary_resource_id: "bbs-trigger"
+        skip_test: true
```

Comment:
Is there a reason we are skipping test here? This prevents it from running in our CI unfortunately

Reply:
Configured 2 reals tests that pass. Done. I thought our API would return an error on creation unless we had some valid bitbucket server resources created. But after checking I realized we can create a trigger with non-existent resource references.